offloading.go

```diff
@@ -187,10 +187,9 @@ func (cm *ContentManager) OffloadContents(ctx context.Context, conts []uint) (in
 		return 0, err
 	}
 
-	// TODO: I believe that we need to hold a lock for the entire period that
-	// we are deleting objects from the blockstore, otherwise a new file could
-	// come in that has overlapping blocks, and have its blocks deleted by this
-	// process.
+	cm.inflightCidsLk.Lock()
```

Comment:
hrm... do we need to hold this over the entire duration? Or can we just lock around the inflight checks and the delete calls individually?